IP fix for System Information tab

sysdrum opened the pull request:
I made the changes I requesting please pull them. I also fixed a typo.

Misko-2083 commented:
Hi @sysdrum ,
when I applied that in linux mint I got several IP's some of them are IPv6
hostname -I will return all IP addresses.
So the problem here is that split is not spliting anything
The shell in Linux Lite must have been confused.
This will work on both Lite and Mint and probably all other:
```python
           ip = execute("hostname -I").split(' ')
           if len(ip) > 1:
              ip = ip[0]
           return ip
```

sysdrum commented:
That is so much cleaner. I will check that into my repo and test it.
On May 30, 2015 10:15 AM, "Милош Павловић" <notifications@github.com> wrote:

> Hi @sysdrum <https://github.com/sysdrum> ,
> when I applied that in linux mint I got several IP's some of them are IPv6
> hostname -I will return all IP addresses.
> So the problem here is that split is not spliting anything
> The shell in Linux Lite must have been confused.
> This will work on both Lite and Mint and probably all other:
>
>            ip = execute("hostname -I").split(' ')
>            if len(ip) > 1:
>               ip = ip[0]
>            return ip
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linuxlite/litecontrolcenter/pull/1#issuecomment-107051662>
> .
>

Misko-2083 commented:
OK,  let us know how it went.

sysdrum commented:
Will do.
On May 30, 2015 1:00 PM, "Милош Павловић" <notifications@github.com> wrote:

> OK, let us know how it went.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linuxlite/litecontrolcenter/pull/1#issuecomment-107072280>
> .
>

linuxlite commented:
@Misko-2083 Great stuff :) That code works here. Will wait for @sysdrum to confirm and I'll merge. Thank you.

Anyone got ideas for picking up the sound adapter in VirtualBox and hardware?

Misko-2083 commented:
@linuxlite  depends on the putput of lspci | grep "Audio|audio"
This is what works with VmWare:
```python
        if info == "audio":
           audio = execute("lspci | grep 'Audio device:'")
	   if len(audio) == 0:
              return  execute("lspci | grep audio").split('controller:')[1].split('(rev')[0].split('[')[0] 
           else:
              return execute("lspci | grep Audio").split('device:')[1].split('(rev')[0].split(',')[0]
```
I need the output of lspci | grep "Audio|audio" on VBox
If the output contains this:
```Multimedia audio controller: ```
The previous code will work
If it's not, we'll have to do something else

sysdrum commented:
I will pull and test that too.
On May 30, 2015 4:20 PM, "Милош Павловић" <notifications@github.com> wrote:

> @linuxlite <https://github.com/linuxlite> depends on the putput of lspci
> | grep "Audio|audio"
> This is what works with VmWare:
>
>         if info == "audio":
>            audio = execute("lspci | grep 'Audio device:'")
>        if len(audio) == 0:
>               return  execute("lspci | grep audio").split('controller:')[1].split('(rev')[0].split('[')[0]
>            else:
>               return execute("lspci | grep Audio").split('device:')[1].split('(rev')[0].split(',')[0]
>
> I need the output of lspci | grep "Audio|audio" on VBox
> If the output contains this:
> Multimedia audio controller:
> The previous code will work
> If it's not, we'll have to do something else
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/linuxlite/litecontrolcenter/pull/1#issuecomment-107087322>
> .
>

Misko-2083 commented:
Thank you @sysdrum 
@linuxlite if you are interested in adding Trash icon on Desktop
```perl
Home
xfconf-query --channel xfce4-desktop --property "/desktop-icons/file-icons/show-home" --set "true"
xfconf-query --channel xfce4-desktop --property "/desktop-icons/file-icons/show-home" --set "false"

Trash show/hide
xfconf-query --channel xfce4-desktop --property "/desktop-icons/file-icons/show-trash" --set "true"
xfconf-query --channel xfce4-desktop --property "/desktop-icons/file-icons/show-trash" --set "false"

Filesystem
xfconf-query --channel xfce4-desktop --property "/desktop-icons/file-icons/show-filesystem" --set "true"
xfconf-query --channel xfce4-desktop --property "/desktop-icons/file-icons/show-filesystem" --set "false"
```
These commands work as if you added them from the Xfce settings and work only on Xfce because this DE is the only one that handles them. You can make a Desktop file for Trash but can't drag and drop files on it and it won't show as full when there is something inside. Same thing with Home. Also when you right-click on Icons, the menu is different.

linuxlite commented:
@Misko-2083 good idea, I'll add the Trash function and look at the other code you've added. Cheers.

linuxlite commented:
@Misko-2083 your audio code works too, fantastic!

linuxlite commented:
Lots of new pushes, please pull before proceeding with further testing. Thank you.

sysdrum commented:
Okay, so I tested the .split(' ') and it works. I tried to merge the commits to one commit but it still wants to push all three changes. Currently my repo is = with main + 3 commits.

sysdrum commented:
Close this pull request, as you have already pushed the code.
